equinor/EchoComponents
# chore: #65365 - Update EDS version to v0.19.0

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description

Updating to EDS v0.19.0 (from 0.17.0). Both version contain breaking changes.

Updated dialog generator component:
- as the Dialog have been changed in the new version

**Other updates**:

Storybook updates:
- DialogGenerator story: Removed wrapper component: it hid the example code in 'Docs' view - now the dev user should see the code example how it's actually used
- Added more proper introduction page

v0.19.0 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.19.0

v0.18.0 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.18.0


### Related PRs:
- EchoFramework: https://github.com/equinor/EchoFramework/pull/207
- EchopediaWeb: https://github.com/equinor/EchopediaWeb/pull/1220

### How to test:
- npm run storybook
- check every component: see if you experience anything wierd
-- focus more on Dialogs and Menus - these should be affected by the breaking changes
